backend/app/helpers/spree/admin/menu_helper.rb

```diff
@@ -47,6 +47,14 @@ def menu_locations_dropdown_values
 
         menu_items_for_select
       end
+
+      def default_menu_for_store?(menu)
```

Comment:
@MatthewKennedy can we just have `default_menu(location)` method on the `Store` model?

Reply:
I'll implement that now.

Reply:
Hi @damianlegawiec 

The highlighted method in the comment above is just a simple view helper method for the admin UI that returns true of false to show if the menu is default in the admin UI Index page.

I added the `default_menu(location)` and used that in the fallback code for the menu, if you want to take a look at the changes?